src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,49 @@
+package calculator;
+
+public class StringCalculator {
+    public static int calc(String input){
+        String[] arr = input.split(" ");
+        int size = arr.length;
+        if(size < 3){
+            throw new IllegalArgumentException();
+        }
+        Operator operator;
+        int num2;
+        int answer = Integer.parseInt(arr[0]);
+        for(int i = 1; i < size - 1; i += 2){
+            operator = Operator.valueOfString(arr[i]);
+            num2 = Integer.parseInt(arr[i+1]);
+            answer = calcFromOperator(operator, answer, num2);
+        }
```

Comment:
```suggestion
        if (size < 3) {
            throw new IllegalArgumentException();
        }
        Operator operator;
        int num2;
        int answer = Integer.parseInt(arr[0]);
        for (int i = 1; i < size - 1; i += 2) {
            operator = Operator.valueOfString(arr[i]);
            num2 = Integer.parseInt(arr[i + 1]);
            answer = calcFromOperator(operator, answer, num2);
        }
```
스페이스 컨벤션도 신경 써보는 것은 어떨까요?

Reply:
이 부분 작성하신 코드 전체적으로 확인 부탁드립니다! 
인텔리제이에 Reformatting 단축키도 있습니다. 
윈도우: Ctrl + Alt + L 
맥 : Cmd + Alt + L

쉽지 않겠지만 조금만 신경써서 코딩하시면 금방 손에 익으실 거라고 생각해요 :)